scripts/prepare-web-pak.js

```diff
@@ -0,0 +1,26 @@
+const fs = require("fs");
+const pak = require("../package.json");
+
+function changePackageName(name) {
+  return JSON.stringify(
+    {
+      ...pak,
+      name: "@draftbit/" + name,
+    },
+    null,
+    2
+  );
+}
+
+function removeDependency(package, dependencyName) {
+  return package.replace(new RegExp(` +"${dependencyName}"[^\n]+\n`), "");
```

Comment:
this is technically json so you could remove the key instead of using regex

Reply:
good call. fixed in 4511140474b858c381ce586bd657c2842d63b0fd